### DISCUSSION TITLE
Please provide an option to save queries.

### DISCUSSION THREAD

**Discussion by sabandi** [community/community#13630] [Other Feature Feedback, Questions, & Ideas] [answered]
I often find my myself searching for something within my org codebase and the filtering few repos out of it. This tasks is repetetive and a feature like `save queries` can help reduce a lot of time.

**Comment by colinwm** [comment, marked as answer]
I would suggest you create a custom scope for your query. Click on the scopes dropdown, and scroll to the bottom to select custom scope. You can even build a scope using boolean operators like NOT, so e.g.

`org:my_org AND NOT (repo:my_org/tests OR repo:my_org/xyz OR ...)`
